app/admin/utilities.rb

```diff
@@ -75,6 +75,24 @@
         end
       end
       column do
+        panel 'Create multiple badge claims' do
+          all_special_badges = Badge.where(tier: 'special').sort_by(&:created_at).reverse
```

Comment:
Thanks for the tip! I agree, it makes more sense to do this database-side. I'll change it.